homework/Repository.js

```diff
@@ -12,22 +12,60 @@ class Repository {
    * Render the repository info to the DOM.
    * @param {HTMLElement} container The container element in which to render the repository.
    */
-  render(container) {
+
+  static render(repository) {
     // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, JSON.stringify(this.repository, null, 2));
+
+    const repositoryContainer = document.getElementById('repository');
+    App.clearContainer(repositoryContainer);
+
+    const h2 = Util.createAndAppend('h2', repositoryContainer, { id: 'repository-name' });
+    Util.createAndAppend('a', h2, {
+      text: repository.name,
+      href: repository.url,
+      target: '_blank',
+    });
+    const ul = Util.createAndAppend('ul', repositoryContainer, { id: 'repository-info' });
+    [
+      'Description',
+      repository.description,
+      'Forks',
+      repository.forks,
+      'Stargazers',
+      repository.stargazers,
+      'Watchers',
+      repository.watchers,
+      'Updated',
+      Date(repository.updatedAt),
+    ].forEach(value => Util.createAndAppend('li', ul, { text: value }));
   }
 
   /**
    * Returns an array of contributors as a promise
    */
-  fetchContributors() {
+  async fetchContributors() {
     return Util.fetchJSON(this.repository.contributors_url);
   }
 
-  /**
-   * Returns the name of the repository
-   */
-  name() {
+  get name() {
     return this.repository.name;
   }
+  get url() {
+    return this.repository.html_url;
+  }
+  get description() {
+    return this.repository.description;
+  }
+  get forks() {
+    return this.repository.forks_count;
+  }
+  get stargazers() {
+    return this.repository.stargazers_count;
+  }
+  get watchers() {
+    return this.repository.watchers_count;
+  }
+  get updatedAt() {
```

Comment:
All of these `get`'s you have in your repository, so if practicing it's okay, but not including in any projects you have in any next homework. :)